tensorflow_lite_support/ios/sources/TFLCommonUtils.h

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#import <Foundation/Foundation.h>
+#import <TensorFlowLiteTaskVisionC/TensorFlowLiteTaskVisionC.h>
```

Comment:
Is this needed?

Reply:
Thanks for the clarification. SGTM.

Reply:
What is "TensorFlowLiteTaskVisionC.h"? It's a little wired to have TFLCommonUtils.h depends on Task Vision C API.